.travis/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:xenial
+
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+       build-essential=12.* \
+       wget=1.17.* \
+       clang-6.0 \
+       g++-multilib=4:5.3.* \
```

Comment:
This should just be `g++`, with no `multilib`. Multilib is for cross-compiling, which we don't want to do.